x-pack/plugins/alerting/server/alert/alert.ts

```diff
@@ -202,6 +209,12 @@ export class Alert<
           meta: {
             flappingHistory: this.meta.flappingHistory,
             flapping: this.meta.flapping,
+            uuid: this.meta.uuid,
+          },
+          state: {
+            ...(this.state.start ? { start: this.state.start } : {}),
+            ...(this.state.duration ? { duration: this.state.duration } : {}),
+            ...(this.state.end ? { end: this.state.end } : {}),
```

Comment:
TODO: think about the implications of this. We are returning this so that we have these values available to write for the AAD document but they will also be serialized to the task document, so maybe we should change how to do this